modules/API/external/external_api.js

```diff
@@ -1131,6 +1131,34 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
      * @returns {void}
      */
     stopRecording(mode) {
-        this.executeCommand('startRecording', mode);
+        this.executeCommand('stopRecording', mode);
+    }
+
+    /**
+     * Sets e2ee enabled/disabled.
+     *
+     * @param {boolean} enabled - The new value for e2ee enabled.
+     * @returns {void}
+     */
+    toggleE2EE(enabled) {
+        this.executeCommand('toggleE2EE', enabled);
+    }
+
+    /**
+     * Sets the key and keyIndex for e2ee.
+     *
+     * @param {Object} keyInfo - The new key and kexIndex.
+     * @returns {void}
+     */
+    setMediaEncryptionKey(keyInfo) {
```

Comment:
Let's use async await here please, and don't catch the export error, this way the caller will know if they have messed up.